src/messageFactory.ts

```diff
@@ -26,6 +26,36 @@ export default class MessageFactory {
     } as NewMessage<MessagePayload>);
   }
 
+  /**
+   * Creates the message for chunk
+   *
+   * @template MessagePayload - the type describing structure of the message payload
+   *
+   * @param type - request type
+   * @param payload - message for chunk
+   * @param chunks - number of chunks
+   */
+  public static createForUpload<MessagePayload>(type: string, payload: MessagePayload, chunks?: number): string {
+    return JSON.stringify({
+      type,
+      payload,
+      chunks,
+      id: MessageFactory.createMessageId(),
+    });
+  }
+
+  /**
+   * Creates the buffer message
+   *
+   * @param fileId - file id
+   * @param bufData - buffer of file and meta data
+   * @param message - message to send with chunk
+   */
+  public static createBufferMessage(fileId: string, bufData: Buffer, message: string): Buffer {
+    const bufMessage = Buffer.from(message);
+    return Buffer.concat( [Buffer.from(fileId), bufData, bufMessage] );
```

Comment:
why `Buffer.from()` for the `bufMessages` moved to the separate variable, but for the `fileId` has not? It would better to move it to the variable too.